Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:alpine
+
+EXPOSE 6667/tcp 6697/tcp 8080/tcp
+
+RUN \
+    apk add --update git && \
+    rm -rf /var/cache/apk/*
+
+RUN mkdir -p /go/src/github.com/oragono/oragono
+WORKDIR /go/src/github.com/oragono/oragono
+
+COPY . /go/src/github.com/oragono/oragono
+
+RUN go get -v -d
+RUN go build .
+
+CMD ["./run.sh"]
```

Comment:
Note, with multi-stage builds now possible, we could make a much smaller runnable container that doesn't have the mess of `.git` and `vendor` laying around.